src/test/java/com/google/ehub/servlets/ItemPageServletTest.java

```diff
@@ -67,6 +70,8 @@
   public void init() {
     MockitoAnnotations.initMocks(this);
     helper.setUp();
+    ProfileDatastore profile = new ProfileDatastore();
+    profile.addUserProfileToDatastore("Eric", EMAIL, USERNAME, "Hey");
```

Comment:
Should `"Eric"` be replaced with `USERNAME`

Reply:
Eric is the Name which differs from the Username

Reply:
Oops thanks for clarifying 👍

Reply:
No problem